packages/osd-ace/package.json

```diff
@@ -16,6 +16,6 @@
     "@osd/babel-preset": "1.0.0",
     "raw-loader": "^4.0.2",
     "typescript": "4.0.2",
-    "webpack": "^4.41.5"
+    "webpack": "npm:@amoo-miki/webpack@4.46.0-rc.2"
```

Comment:
From my understanding, in webpack 4, the `md4` hashing algorithm was hardcoded into the system. This means it was the default and unchangeable way that Webpack 4 created hash digests, which are essential for long-term caching of the assets. This hardcoding might have also extended to some plugins, leading to potential compatibility or performance issues. We need to pass `--openssl-legacy-provider` flag to use older or deprecated features or behavior.

This patched Webpack `amoo-miki/webpack` is to use `xxhash64`, which is a different hashing algorithm. `xxhash64` is known to be a very fast non-cryptographic hash algorithm, thus potentially improving performance. This is used in Webpack 5. 

Since bumping to Webpack 5 is difficult for us now due to older versions of other tools, using a patched webpack  provides us the benefits of using `xxhash64` hash function without having to update to Webpack 5. This strategy allows us separating updates of Node.js from updates of Webpack. 

I will summarize it into the description of this PR.

Reply:
Perfect, that explanation is great.